### ISSUE TITLE
OpenAPI spec differs from response on /control/dhcp/interfaces

### ISSUE THREAD

**Issue by jmaister** [AdguardTeam/AdGuardHome#7510]
### Prerequisites

- [X] I have checked the [Wiki](https://github.com/AdguardTeam/AdGuardHome/wiki) and [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions/categories/q-a) and found no answer

- [X] I have searched other issues and found no duplicates

- [X] I want to report a bug and not [ask a question or ask for help](https://github.com/AdguardTeam/AdGuardHome/discussions/categories/q-a)

- [X] I have set up AdGuard Home correctly and [configured clients to use it](https://github.com/AdguardTeam/AdGuardHome/wiki/Clients). (Use the [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions/categories/q-a) for help with installing and configuring clients.)

### Platform (OS and CPU architecture)

Windows, AMD64 (aka x86_64)

### Installation

Docker

### Setup

On one machine

### AdGuard Home version

AdGuard Home, version v0.107.53

### Action

Replace the following command with the one you're calling or a
description of the failing action:

```sh
C:\Users\jordi>curl -u USER:PASS -X GET http://localhost/control/dhcp/interfaces
```

### Expected result

The response differes from the OpenAPI spec defined in the project: https://github.com/AdguardTeam/AdGuardHome/blob/master/openapi/openapi.yaml

The current result based on OpenAPI returns a list of `ip_addresses` but the API returns `ipv4_addresses` and `ipv6_addresses`:

Also, the `gateway_ip` field is missing in the spec.

```
    'NetInterface':
      'type': 'object'
      'description': 'Network interface info'
      'required':
      - 'flags'
      - 'hardware_address'
      - 'name'
      - 'mtu'
      'properties':
        'flags':
          'type': 'string'
          'description': >
            Flags could be any combination of the following values, divided by
            the "|" character: "up", "broadcast", "loopback", "pointtopoint" and
            "multicast".
          'example': 'up|broadcast|multicast'
        'hardware_address':
          'type': 'string'
          'example': '52:54:00:11:09:ba'
        'name':
          'type': 'string'
          'example': 'eth0'
        'ip_addresses':
          'type': 'array'
          'items':
            'type': 'string'
        'mtu':
          'type': 'integer'
```

### Actual result

{"eth0":{"name":"eth0","hardware_address":"02:42:ac:14:00:02","flags":"up|broadcast|multicast|running","gateway_ip":"172.20.0.1","ipv4_addresses":["172.20.0.2"],"ipv6_addresses":null}}

**Comment by ainar-g** [comment]
@EugeneOne1, please investigate.

**Comment by EugeneOne1** [comment]
@jmaister, hello and thank you for bringing this to our attention.  We've updated the `openapi/openapi.yaml` specification, it's available since 261c1599a528f2e00b428f2ffef3f73c68d01e27.
